### PR TITLE
#152002813 Allow sync over of events addresses

### PR DESCRIPTION
**Pull request by Wardormeur**
add entity for event save
add controller function for saving addresses
remove old tests (lol)
add migration for next events
add entity:list for v_next_events
add useDojoAddress to save the configuration of the event

### CODE REVIEW

File: lib/entity/event/save.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = function (args, done) {
+  var seneca = this;
+  var event = args.event;
+  seneca.make$('cd/events').save$(event);
```
Comment (DanielBrierton) [inline lib/entity/event/save.js]: Missing call to done.

File: lib/controllers/event/update-address.spec.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    lab.afterEach(function (done) {
+        sandbox.restore();
+        done();
+    });
+
+    lab.test('should get next events and update addresses of those events', function (done) {
+        // ARRANGE
+        var dojoId = 1;
+        var mockLocation = { address: 'aha',
```
[the remark this answers is not in the document]
Comment (Wardormeur) [reply lib/controllers/event/update-address.spec.js]: For some reason, i'm allergic to jump of lines on the first and last lines, expect if it's a function

File: lib/controllers/event/update-address.spec.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            });
+          cb(null, mockEvents);
+      });
+      // ACT
+      updateAddress({role: 'cd-events', dojoId: 1, location: mockLocation}, function (err, ret) {
+        expect(err).to.be.eql(undefined);
+        expect(ret).to.be.eql(undefined);
+        expect(senecaStub.act
+          .withArgs(sinon.match({ role: 'cd-events', entity: 'event', cmd: 'save'}))
+        ).to.not.have.been.called;
```
[the remark this answers is not in the document]
Comment (Wardormeur) [reply lib/controllers/event/update-address.spec.js]: Took some time to figure something when act is called twice. i think that's the cleanest i could find

File: lib/entity/event/save.js (hunk 1 above)
Comment (Wardormeur) [reply lib/entity/event/save.js]: This should have been caught by tests if i've handled doneSpy.to.have.been.calledOnce;

**Pull request merged**